src/main/haskell/language-kore/src/Data/Kore/ASTUtils/SmartConstructors.hs

```diff
@@ -208,13 +220,13 @@ pattern Var_
     :: var level
     -> PureMLPattern level var
 
--- pattern StringLiteral_
---   :: StringLiteral
---   -> Fix (Pattern level var)
+pattern StringLiteral_
+  :: StringLiteral
+  -> PureMLPattern Meta var
```

Comment:
Why doesn't StringLiteral_ and CharLiteral_ match direclty the Sting / Char they wrap?